File: 240304/개발자의 능력 3/ability-of-developer-3.py
import sys
number=list(map(int,input().split()))

def get_diff(i,j,k):
    sum1=number[i]+number[j]+number[k]
    sum2=sum(number)-sum1 
    return(abs(sum1-sum2))

ans=sys.maxsize
for i in range(6):
    for j in range(i+1,6):
        for k in range(j+1,6):
            ans=min(ans,get_diff(i,j,k))


print(ans)

File: 260110/개발자의 능력 3/ability-of-developer-3.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');
const arr = input[0].trim().split(' ').map(Number);

// Please Write your code here.
let sum = 0;
arr.forEach((number)=>{
    sum+=number;
})

let ans = Number.MAX_SAFE_INTEGER;

for(let i=0; i<6; i++){
    for(let j=i+1; j<6; j++){
        for(let k=j+1; k<6; k++){
            let cnt = arr[i]+arr[j]+arr[k];
            ans = Math.min(ans, Math.abs(cnt-Math.abs(sum-cnt)))
        }
    }
}

console.log(ans)